ppx/browser/ppx_deriving_json_runtime.ml

```diff
@@ -3,8 +3,23 @@ type t = Js.Json.t
 let to_json t = t
 let of_json t = t
 let to_string t = Js.Json.stringify t
-let of_string s = Js.Json.parseExn s
-let of_json_error msg = raise @@ Json.Decode.DecodeError msg
+
+exception Json_error of string
```

Comment:
encode_error?

Reply:
no, this is JSON parsing error, not encode

Reply:
Parsing_error then? ^^

Reply:
Or maybe `Of_string_error` or `String_to_json_error`. I find a bit confusing that this is called `Json_error` and the errors happening at the later stage are called `Of_json_error`.

Reply:
Fixed to be `Of_string_error`.